### ISSUE TITLE
Weird results

### ISSUE THREAD

**Issue by kuh-muh** [memphis-iis/cohmetrix-issues#108]
Hi, 
Im using the desktop version and after running the tool, i get results, that are very different than those from the web tool. I also get only 68 indecies. Im not sure why this is happening as im using the same text.

**Comment by JRustyHaner** [comment]
@kuh-muh could you email the corpus and results to iisdevs@memphis.edu and I'll check on it personally.

**Comment by kuh-muh** [comment]
@JRustyHaner is the mail adress correct? i received the message that the email could not be delivered

**Comment by JRustyHaner** [comment]
@kuh-muh try jrhaner@memphis.edu. Sorry, we're in the holiday season

**Comment by kuh-muh** [comment]
@JRustyHaner  have you checked it yet?

**Comment by mitanshu7** [comment]
Yes, when using the cohmetrix-cli on linux, i only get 70 indices too, as opposed to the promised 150.
